playbook.py

```diff
@@ -0,0 +1,486 @@
+"""
```

Comment:
Can we rename this file to `code42_alert_response_playbook.py`

Reply:
Actually, I think this file is in here twice. This one can be deleted?  And the other one that is named `"Peter ..."`
should have the name above